src/files.js

```diff
@@ -310,6 +310,12 @@ FOAM_FILES([
   { name: "foam/nanos/logger/Logger" },
   { name: "foam/nanos/logger/DAOLogger" },
   { name: "foam/nanos/logger/LoggingDAO" },
+
+  { name: "foam/nanos/notification/Notifiable" },
+  { name: "foam/nanos/notification/EmailSetting" },
+  { name: "foam/nanos/notification/NotificationSetting" },
```

Comment:
Is it really necessary to add these three classes? They have nothing to do with this PR.